app/templates/client/components/auth(auth)/auth.service.js

```diff
@@ -20,7 +19,7 @@ function AuthService($location, $http, $cookies, $q, appConfig, Util, User) {
      * @param  {Function} callback - optional, function(error, user)
      * @return {Promise}
      */
-    login: function(user, callback) {
+    login: function(user, callback = angular.noop) {
```

Comment:
So to use the promise behavior a user would have to run `Auth.getCurrentUser(function() {}).then();` ? Why not just `Auth.getCurrentUser(true).then();` ?